internal/manifests/gateway/gateway.go

```diff
@@ -168,6 +168,8 @@ func deployment(params manifestutils.Params, rbacCfgHash string, tenantsCfgHash
 								fmt.Sprintf("--traces.write.endpoint=%s:4317", naming.ServiceFqdn(tempo.Namespace, tempo.Name, manifestutils.DistributorComponentName)),
 								fmt.Sprintf("--traces.read.endpoint=%s://%s:16686", httpScheme(params.Gates.HTTPEncryption),
 									naming.ServiceFqdn(tempo.Namespace, tempo.Name, manifestutils.QueryFrontendComponentName)),
+								fmt.Sprintf("--traces.tempo.endpoint=%s://%s:3200", httpScheme(params.Gates.HTTPEncryption),
```

Comment:
Let's use `manifestutils.PortHTTPServer` for the Tempo port, and also the relevant port constant from `manifestutils/constants.go` for the 2 other hardcoded ports above.